graphql/resolvers/session.ts

```diff
@@ -45,7 +45,7 @@ export const session = async (_parent: void, _args: void, context: Context) => {
     discordUserId,
     discordAvatarUrl,
     discordUsername,
-    isConnectedToDiscord: !!user.discordRefreshToken // using this to avoid a second fetch to get Discord username
+    isConnectedToDiscord: !!user.discordId // using this to avoid a second fetch to get Discord username
```

Comment:
It has been changed to `discordId` because with how next-auth work, we can't put sensitive data such as the user Discord refresh token in the session.